### ISSUE TITLE
Arm64 native client cannot parse tflite pre-trained-model

### ISSUE THREAD

**Issue by isjosan** [mozilla/DeepSpeech#2262]
I ran the native_client.v0.6.0-alpha.4.tflite binary on Ubuntu Linux hosted on HP desktop; with the pre-trained tflite model from v0.5.0 and it did inference successfully from the sample audio file:

**$ ~/Downloads/tflite_native_client/deepspeech --model ./output_graph.tflite --alphabet ./alphabet.txt --lm ./lm.binary  --audio ../audio/4507-16021-0012.wav**
TensorFlow: v1.14.0-14-g1aad02a
DeepSpeech: v0.6.0-alpha.4-0-gdf2a1d6
INFO: Initialized TensorFlow Lite runtime.
why should one halt on the way

However, when I run native_client.v0.6.0-alpha.4.arm64  binary on Mendel Linux hosted on Coral Edge Arm64 Dev board; with the pre-trained tflite model from v0.5.0; it fails with the following error:

**$ ../arm64_native_client/deepspeech --model ./output_graph.tflite --alphabet ./alphabet.txt --lm ./lm.binary  --audio ../audio/4507-16021-0012.wav**
TensorFlow: v1.14.0-14-g1aad02a
DeepSpeech: v0.6.0-alpha.4-0-gdf2a1d6
Warning: reading entire model file into memory. Transform model file into an mmapped graph to reduce heap usage.
**Data loss: Can't parse ./output_graph.tflite as binary proto
Could not create model.**

**It appears that I need an arm64-tflite specific binary to parse the tflite model; Any suggestions where I can get that? Is there a way I can generate arm64-tflite specific binary to parse the tflite model?**

**Comment by lissyx** [comment]
I've already explained that to you with links in issue #2248
